packages/fe-fpm-writer/src/controller-extension/index.ts

```diff
@@ -243,6 +243,11 @@ export function generateControllerExtension(
 
     if (controllerConfig.typescript) {
         addExtensionTypes(basePath, controllerConfig.minUI5Version, fs);
+        // Typescript - add declaration ts file for 'sap/ui/core/mvc/ControllerExtension', which allows access to `this.base.getExtensionAPI()` within Controller extension classes
+        const declarationFilePath = join(basePath, '/webapp/ext/controller/ControllerExtension.d.ts');
```

Comment:
Just wondering, do we want to put the type in ext/controller/ControllerExtension or in another place only for type extensions ? 
Shouldn't we otherwise offer this definition in a type module that we deliver. (and that we can easily tweak ?)

Reply:
This is a workaround since there is no such types module. Let's have a discussion with the SAP Fiori elements architects and find out if we can release something like that.